File: backend/app/models/pvp.py
from pydantic import BaseModel


class PvpActivity(BaseModel):
    activityHash: int
    activityDirectorHash: int
    standing: int
    kills: int
    deaths: int
    assists: int
    killsDeathsRatio: float
    killsDeathsAssists: float
    efficiency: float
    period: str

File: frontend/src/domain/types/pvp.ts
export interface PvpActivity {
  activityHash: number;
  activityDirectorHash: number;
  standing: number;
  kills: number;
  deaths: number;
  assists: number;
  killsDeathsRatio: number;
  killsDeathsAssists: number;
  efficiency: number;
  period: string;
}
